examples/chip-tool/commands/common/DeviceScanner.h

```diff
@@ -50,6 +51,7 @@ class DeviceScanner : public chip::Dnssd::CommissioningResolveDelegate,
 {
 public:
     CHIP_ERROR Start();
+    CHIP_ERROR Start(chip::Optional<uint64_t> compressedFabricId);
```

Comment:
Please document what those do (commissionable discovery, vs operational discovery optionally restricted to a specific fabric).

Reply:
I have renamed those to `StartCommissionableDiscovery` and `StartOperationalDiscovery`. I was sort of hoping that the 'optional' nature of the argument was self-explanatory. I renamed it 'compressedFabricIdFilter'. Is that enough ?